src/redux/messenger/selector.ts

```diff
@@ -0,0 +1,25 @@
+import { RootState } from '../store';
+import { createSelector } from '@reduxjs/toolkit';
+
+const selector = (state: RootState) => state.messenger;
+
+export const chatUsers = createSelector(
+  [selector],
+  (state) => state.messenger.chatUsers
+);
+
+export const conversations = createSelector(
+  [selector],
+  (state) => state.messenger.conversations
+);
+
+export const activeChatID = createSelector(
+  [selector],
+  (state) => state.messenger.activeChatID
+);
+
```

Comment:
Why export them from here if we will use them via appSelector?
```suggestion
const chatUsers = createSelector(
  [selector],
  (state) => state.messenger.chatUsers
);

const conversations = createSelector(
  [selector],
  (state) => state.messenger.conversations
);

const activeChatID = createSelector(
  [selector],
  (state) => state.messenger.activeChatID
);

```

Reply:
done